src/openloco/windows/tooltip.cpp

```diff
@@ -235,4 +235,9 @@ namespace openloco::ui::tooltip
             _tooltipNotShownTicks = 0;
         }
     }
+
+    void set_52336E(uint8_t value)
```

Comment:
I'm only seeing xrefs to this being set to 0 or 1, so I think we should use `bool` for this.

Reply:
Should we then change also _52336E variable to bool?

Reply:
No we will have to provide a conversion. As otherwise functions we haven't implemented might not like it.

Reply:
I checked the xrefs to _52336E and only saw `mov` and `cmp` instructions involving 0 or 1. Am I missing something?

Reply:
I wasn't 100% sure if true is defined to have the value 1. It looks like it does so this would actually be fine.

Reply:
Ok, so are we redefining also _52336E to bool?

Reply:
Yes i think that will work fine